infrastructure/helm/quantumserverless/values.yaml

```diff
@@ -76,7 +95,7 @@ ray-cluster:
   head:
     initArgs:
       dashboard-host: '0.0.0.0'
-      port: '6379'
```

Comment:
I suggest do that in opposite way. As ray is our 'more important' component we should leave it as is, otherwise we have to configure it everywhere in client. While redis is supplementary dependency, which can be turned off. 

What do you think?

Reply:
My way to think here was: "How the dashboard even being from Ray is maybe something secondary  I can change it". But I don't have a strong opinion honestly. I'm fine with your approach too!

Reply:
oh, I know the confusion :) Dashboard is a strange name, but actually it is entrypoint to connect to ray head node (GCS server). So if ray need to do some distributed computation, you usually specify 'auto' connection in context initialization and it will connect to that port. I think, the reason for this port is exactly 6379 because they used redis for that purpose

Reply:
Ooooh, I see. Thanks for the clarification, I will change it then!